src/main/java/edu/ie3/datamodel/io/extractor/Extractor.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * © 2020. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.datamodel.io.extractor;
+
+import edu.ie3.datamodel.exceptions.ExtractorException;
+import edu.ie3.datamodel.models.input.InputEntity;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * //ToDo: Class Description
+ *
+ * @version 0.1
+ * @since 31.03.20
+ */
+public class Extractor {
```

Comment:
I would suggest to have a simple static class. So then you could save the instantiation.

```suggestion
public class NestedEntityUtils {

public NestedEntityUtils() {
    throw new IllegalStateException("Do not instantiate static utility class");
}
```